apps/pano/src/features/post/MoreOptionsDropdown.tsx

```diff
@@ -32,6 +32,9 @@ const DotsButton = styled(IconButton, {
 
 const Item = styled(DropdownMenuItem, {
   minWidth: 0,
+  "&:hover": {
+    cursor: "pointer",
+  },
```

Comment:
Let's revert this, menu interactions don't need to have `cursor: pointer`

Reply:
done